frontend/src/pages/eventoverview.js

```diff
@@ -0,0 +1,174 @@
+      description: this.state.description
+    }
+    await createEvent(event)
+    this.closeModal()
+    Router.push({ pathname: '/eventoverview' })
+  }
+
+  render() {
+    return (
+      <>
+        <Nav />
+        <div className="page-content-wrapper">
+          <Container fluid>
+            <Modal isOpen={this.state.modal}>
+              <ModalHeader>Add New Event</ModalHeader>
+              <ModalBody>
+                <Form>
+                  <FormGroup>
+                    <Label>Name</Label>
+                    <Input
+                      name="name"
+                      placeholder="i.e. Product Showcase"
+                      onChange={this.handleChange}
+                    />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Date</Label>
+                    <Input name="date" placeholder="i.e. 06/20/19" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Start Time</Label>
+                    <Input name="startTime" placeholder="i.e. 5" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>End Time</Label>
+                    <Input name="endTime" placeholder="i.e. 6" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Location</Label>
+                    <Input name="location" placeholder="i.e. ECEB" onChange={this.handleChange} />
+                  </FormGroup>
+                  <FormGroup>
+                    <Label>Description</Label>
+                    <Input
+                      name="description"
+                      placeholder="i.e. Teams present their final product."
+                      onChange={this.handleChange}
+                      onKeyPress={this.handleKeyPress}
+                    />
+                  </FormGroup>
+                </Form>
+              </ModalBody>
+              <ModalFooter>
+                <Button color="secondary" onClick={this.closeModal}>
+                  Cancel
+                </Button>
+                <Button color="primary" onClick={this.addEvent}>
+                  Submit
+                </Button>
+              </ModalFooter>
+            </Modal>
+            <Table>
+              <thead>
+                <ActionButton
+                  style={{ marginBottom: 15 }}
+                  text="Add New Event"
+                  onClick={this.openModal}
+                />
+                <tr>
+                  <th>Name</th>
+                  <th>Date</th>
+                  <th>Start Time</th>
+                  <th>End Time</th>
+                  <th>Location</th>
+                  <th>Attendees</th>
+                  <th>Facebook Event Link</th>
+                  <th />
+                </tr>
+              </thead>
+              <tbody>
+                {this.state.events.map(event => (
```

Comment:
This map might give a unique key error - in general i think React needs unique keys so that it can perform minimal rendering/rerendering. Disregard if it doesn't, but definitely take a look at this